demos/audio/README.md

```diff
@@ -180,3 +208,95 @@ print(transcript.text)
+Here is an example of OpenVINO/whisper-large-v3-fp16-ov deployment:
+
+:::{dropdown} **Deploying with Docker**
+
+Select deployment option depending on how you prepared models in the previous step.
+
+**CPU**
+
+Running this command starts the container with CPU only target device:
+```bash
+mkdir -p models
+docker run -d -u $(id -u):$(id -g) --rm -p 8000:8000 -v $(pwd)/models:/models:rw openvino/model_server:latest --rest_port 8000 --source_model OpenVINO/whisper-large-v3-fp16-ov --model_repository_path /models --model_name OpenVINO/whisper-large-v3-fp16-ov --task speech2text
+```
+**GPU**
+
+In case you want to use GPU device to run the generation, add extra docker parameters `--device /dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1)`
+to `docker run` command, use the image with GPU support.
+It can be applied using the commands below:
+```bash
+mkdir -p models
+docker run -d -u $(id -u):$(id -g) --rm -p 8000:8000 --device /dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) -v $(pwd)/models:/models:rw openvino/model_server:latest-gpu --rest_port 8000 --source_model OpenVINO/whisper-large-v3-fp16-ov --model_repository_path models --model_name OpenVINO/whisper-large-v3-fp16-ov --task speech2text --target_device GPU
+```
+:::
+
+:::{dropdown} **Deploying on Bare Metal**
+
+If you run on GPU make sure to have appropriate drivers installed, so the device is accessible for the model server.
+
+```bat
+mkdir models
+ovms --rest_port 8000 --source_model OpenVINO/whisper-large-v3-fp16-ov --model_repository_path models --model_name OpenVINO/whisper-large-v3-fp16-ov --task speech2text --target_device CPU
+```
+or
+```bat
+ovms --rest_port 8000 --source_model OpenVINO/whisper-large-v3-fp16-ov --model_repository_path models --model_name OpenVINO/whisper-large-v3-fp16-ov --task speech2text --target_device GPU
+```
+:::
+
+### Request Generation 
+Transcript and translate file that was previously generated with audio/speech endpoint.
+
+:::{dropdown} **Unary call with curl**
+
+
+```bash
+curl http://localhost:8000/v3/audio/translations -H "Content-Type: multipart/form-data" -F file="@speech_spanish.wav" -F model="OpenVINO/whisper-large-v3-fp16-ov"
+```
+```json
+{"text": " Madrid is the capital of Spain."}
+```
+:::
+
+:::{dropdown} **Unary call with python OpenAI library**
+
+```python
+from pathlib import Path
+from openai import OpenAI
+
+filename = "speech_spanish.wav"
+url="http://localhost:8000/v3"
+
+
+speech_file_path = Path(__file__).parent / filename
+client = OpenAI(base_url=url, api_key="not_used")
+
+audio_file = open(filename, "rb")
+translation = client.audio.translations.create(
+  model="OpenVINO/whisper-large-v3-fp16-ov",
+  file=audio_file
+)
+
+print(translation.text)
+```
+```
+Madrid is the capital of Spain.
+```
+:::
+> **Note:** For now speech translation requests are processed synchronously.
```

Comment:
Where do we document limitations? For example text2speach currently works on CPU only. Streaming is not supported yet.